<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>launcherSettings</class>
 <widget class="QDialog" name="launcherSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>402</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>402</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки лаунчера</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Запуск</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="launch_label">
        <property name="text">
         <string>Действие после запуска игры</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="launchBox">
        <item>
         <property name="text">
          <string>Ничего</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Свернуть лаунчер</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Закрыть лаунчер</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Стиль</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="style_label">
          <property name="text">
           <string>Цвет интерфейса</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="styleBox">
          <item>
           <property name="text">
            <string>Стандартный</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Темный</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="documentMode">
        <property name="text">
         <string>Фон и контур во вкладках</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Слежение за серверов</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Обновление каждые</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="updateTimeSpin">
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>мсек</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Уведомлять если статус</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="stateBox">
          <item>
           <property name="text">
            <string>Любой</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Игра</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Брифинг</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Лобби</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Создание</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Обновления</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkUpdateAfterStart">
        <property name="text">
         <string>Проверять обновления при запуске лаунчера</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="launchCheckUpdate">
        <property name="text">
         <string>Запустить проверку обновлений</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>launcherSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>launcherSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
